### PR TITLE
feat: make test harness a little more dynamic

### PR DESCRIPTION
**Pull request by garethjevans**
Fake jenkins will record the jobs triggered and their parameters so this information can be included in the build output

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -143,6 +174,15 @@ func JobInfoHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if name == "job-with-source-parameters" {
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		body := fmt.Sprintf(jobInfoWithSourceParameters, srv.Addr)
+		fmt.Println(body)
+		fmt.Fprint(w, body)
```
Comment (rszumlakowski) [inline main.go:182]: Why do you have `fmt.Println(body)` just to run it through `Fprintf` right after?  Is it just to add a newline to the end of the body?  You don't really need it, right?

Comment (garethjevans) [reply main.go:182]: `fmt.Println(body)` writes the body to stdout , `fmt.Fprint(w, body)` writes the body to the writer (http response)

Comment (rszumlakowski) [reply main.go:182]: Oh, of course.  Sorry, I should have realized that.  :/

**Review by rszumlakowski — APPROVED** [review]
Looks good to me

**Pull request merged**